assets/pages/contributions-landing/reducers/reducers.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { combineReducers } from 'redux';
+
+import { abTestReducer as abTests } from 'helpers/abtest';
```

Comment:
As far I understand we are not defining a new test in this PR and there is no test defined in the ABTest helper. Therefore, probably you shouldn't include this Reducer here. If you decide to remove it, double check `PageStartUp` if it still works.